community/modules/scheduler/htcondor-configure/variables.tf

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+  description = "Project in which HTCondor pool will be created"
+  type        = string
+}
+
+variable "deployment_name" {
```

Comment:
Historically, this evolved from a module outside the Toolkit with a variable `pool_name`.

The best working definition of an HTCondor Pool is the Central Manager and the set of Execute Points configured so that the Central Manager will trust them. The Execute Points can be extremely heterogeneous, distributed across large network gaps, and are, strictly speaking, independent of one another and of the Central Manager. So although people talk about "Condor clusters", the preferred nomenclature is "pool" because it's a little more nebulous. Water comes and goes.

That said, I don't think HTCondor has a distinct configuration setting for a pool name. Basically, you say "who is my central manager?" and that defines the pool to which you belong or are submitting jobs to.

So, I think I should update the description here to align with other uses of deployment name and indicate that cloud resources will receive this name. Leave HTCondor out of it a bit.

If my understanding above is flawed, we can add `pool_name` that takes on some default derived from `deployment_name`.

Reply:
Should be resolved.

Reply:
I am left wondering how this would work if I am defining 2 Pools in the same blueprint. Is that something that would ever happen? In that case would one be inclined to overwrite deployment_name? That is probably behavior we would like to avoid.

While interested in the above, I am also fine with it as is. If there is value in adding a pool_name later that defaults to deployment name, then that should not be a breaking change.